src/test/miner_tests.cpp

```diff
@@ -307,6 +308,7 @@ BOOST_AUTO_TEST_CASE(CreateNewBlock_validity) {
     static_assert(sizeof(blockinfo) / sizeof(*blockinfo) == 110,
                   "Should have 110 blocks to import");
     int baseheight = 0;
+    return;
```

Comment:
This will be removed in the final version?

Reply:
Yes. I don’t want to rent GPUs every time we tweak this PR.

Reply:
Done